Resources/Prototypes/Entities/Objects/Specific/syndicate.yml

```
@@ -148,10 +148,10 @@
   components:
   - type: Store
     balance:
-      Telecrystal: 60
+      Telecrystal: 80 #Sunrise edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)